stablehlo/transforms/StablehloLegalizeQuantizedOpToQDQ.cpp

```diff
@@ -109,27 +109,28 @@ class StablehloLegalizeQuantizedOpToQDQPass
 
 template <typename... StablehloOpTypes>
 void populateStablehloLegalizeQuantizedOpToQDQPatterns(
-    RewritePatternSet* patterns, MLIRContext* context) {
-  patterns->add<QuantizedStablehloOpConversion<StablehloOpTypes>...>(context);
+    RewritePatternSet* patterns, MLIRContext* context, PatternBenefit benefit) {
+  patterns->add<QuantizedStablehloOpConversion<StablehloOpTypes>...>(context,
+                                                                     benefit);
 }
 
 }  // namespace
 
 void populateStablehloLegalizeQuantizedOpToQDQPatterns(
-    RewritePatternSet* patterns, MLIRContext* context) {
+    RewritePatternSet* patterns, MLIRContext* context, PatternBenefit benefit) {
   // The following list covers most of the operations which, according to the
   // stablehlo spoecification document, interprets the quantized
   // operation using dequant-op-quant strategy. The ones excluded are
-  // AddOP, ConvolutionOp, DotGeneralOp, and DynamicConvOp, which are current
+  // ConvolutionOp, DotGeneralOp, and DynamicConvOp, which are current
```

Comment:
We're using benefit to manage this now, can we delete this comment and add these ops into the pass + tests? (Can be done in a followup)

Reply:
make sense. Remove the comment. Will make sure any ops added to the list has the requisite test-cases.